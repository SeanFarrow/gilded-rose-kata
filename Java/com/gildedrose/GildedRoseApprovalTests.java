package com.gildedrose;

import org.junit.Test;
import org.approvaltests.Approvals;
import org.approvaltests.reporters.UseReporter;
import org.approvaltests.reporters.JunitReporter;

@UseReporter(JunitReporter.class)
public class GildedRoseApprovalTests {

    @Test
    public void testUpdateQuality() {
        Item expectedItem =new Item("foo", -1, 0);
    	Item[] items = new Item[] { new Item("foo", 0, 0) };
        GildedRose app = new GildedRose(items);
        app.updateQuality();
        Approvals.verify(expectedItem);
    }

}
