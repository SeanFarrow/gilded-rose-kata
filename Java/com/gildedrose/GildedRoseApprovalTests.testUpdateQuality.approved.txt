[foo, 0, 0] => foo, -1, 0 
[+5 Dexterity Vest, 0, 0] => +5 Dexterity Vest, -1, 0 
[Aged Brie, 0, 0] => Aged Brie, -1, 2 
